### DISCUSSION TITLE
Slicer Alpha Version Packages

### DISCUSSION THREAD

**Discussion by tristangobin** [HALRobotics/Beta#93] [Additive Manufacturing Processes]
[hal.mcneel.v5.cam.1.3.2-beta.zip](https://github.com/HALRobotics/Beta/files/6212188/hal.mcneel.v5.cam.1.3.2-beta.zip)

[CAMDemoFile.zip](https://github.com/HALRobotics/Beta/files/6212298/CAMDemoFile.zip)

**Comment by Thomas-Delaplanche** [comment]
[hal.mcneel.v5.cam.1.3.6-beta.zip](https://github.com/HALRobotics/Beta/files/6459070/hal.mcneel.v5.cam.1.3.6-beta.zip)

**Comment by tristangobin** [comment]
[hal.mcneel.v5.cam.1.4.2-beta.zip](https://github.com/HALRobotics/Beta/files/7527932/hal.mcneel.v5.cam.1.4.2-beta.zip)
